content/posts/blog/2021-01-24_blog_InstantMessagingEsgehtnichtumdieApp

```diff
@@ -0,0 +1,62 @@
+
+Viele fügen sogar einen weiteren Messenger zu ihrer immer größer werdenden Sammlung von Messengern hinzu, was wahrscheinlich eher frustrierend als hilfreich ist. Das bringt mich zurück zur ursprünglichen Fragestellung: Welches Problem will man hier lösen? Was sind die Absichten? Kann man eine Lösung mit einer besseren technologischen Basis finden, 
+die den Hype vermeidet und bei der man nicht mehrere Messenger-Apps installieren muss?
+
+Verschiedene Menschen haben verschiedene Überlegungen zur Beantwortung dieser Frage - manche benötigen einen effektiven Datenschutz, oder Datenhoheit, oder einfach die Möglichkeit, alle ihre Kontakte von einem Ort aus zu erreichen. Der Wechsel von einem Messaging-System zu einem anderen bedeutet jedoch oft, einige Kontakte zurückzulassen. Viele Messaging-Systeme erfordern außerdem eine Telefonnummer, was für den Datenschutz nicht gerade förderlich ist.
+
+### Souveränität der Kommunikation
+
+Wie Sie vielleicht schon vermutet haben, werde ich für XMPP plädieren. Ich denke, die erste zu treffende Entscheidung sollte nicht sein, welchen Messenger man verwendet, sondern welche zugrundeliegende Software-Technologie. Betrachten Sie zuerst die Wahl der Technologie, bevor Sie von einer Empfehlung zur nächsten springen.
+
+> XMPP ist ein offenes Protokoll, so wie HTTP für das Web ist. Es spielt keine Rolle, wie Ihre Website aussieht, jeder kann mit ihr interagieren. Dieses Prinzip ist die Idee hinter XMPP, allerdings für Instant Messaging.
+
+Bei der XMPP Standards Foundation und bei vielen anderen, die an dieser offenen Technologie beteiligt sind, sind wir der Meinung, dass XMPP als Kommunikationstechnologie eine gute Wahl ist, nicht nur im Hinblick auf den Datenschutz. XMPP gibt es seit über zwanzig Jahren, und in dieser Zeit wurde eine Menge praktischer Erfahrung gesammelt. Dazu gehören viele wichtige Indikatoren für die Wahl der Technologie und die Ermöglichung von Souveränität der Kommunikation; die wichtigen Indikatoren sind:
+
+- Dezentralisierung von Kommunikationsdiensten (Föderation)
+- Standardisierung und Erweiterbarkeit der Technologie
+- Interoperabilität
+- Innovation und den Einsatz offener Entwicklung
+- Privatsphäre und Kontrolle, auch durch den Einsatz von Ende-zu-Ende-Verschlüsselung
+
+> XMPP stellt definierte Informationen zur Verfügung, wie die Kommunikation in einem Netzwerk abgewickelt werden soll. Der Anwender entscheidet, welche Messaging-Software passt oder wo die Daten gespeichert werden sollen.
+
+Im Gegensatz zur aktuellen Situation, in der andere Ihnen ihren bevorzugten Messenger vorschlagen, haben Sie im XMPP-Universum die freie Wahl aus vielen Messengern - suchen Sie sich den aus, der Ihnen gefällt, nicht den, den andere mögen 
```

Comment:
Situation, in der jeder seinen bevorzugten Messenger vorschlägt, hat man im XMPP-Universum [...] Jeder kann den Messenger wählen der ihm gefällt, nicht den den andere einem vorgeben, bis dann die nächste Trend-App erscheint.